common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -64,6 +65,9 @@
             WORKDIR = tmpdir();
             logger.debug("-Dio.netty.native.workdir: " + WORKDIR + " (io.netty.tmpdir)");
         }
+
+        DELETE_NATIVE_LIB_AFTER_LOADING = SystemPropertyUtil.getBoolean(
+                "io.netty.native.deleteLibAfterLoading", true);
```

Comment:
should this be scoped to a particular library? Just wondering if we ever load multiple libraries using this loader if we want to have a global flag or scope it per load.